### PR TITLE
Melnyk/rvat 725 726 verify title of dropdown sub menu events

### PR DESCRIPTION
**Pull request by MelnykNikita**

### CODE REVIEW

**Review by TyshchenkoRoma — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
 		<dependency>
 			<groupId>org.seleniumhq.selenium</groupId>
 			<artifactId>selenium-java</artifactId>
-			<version>2.45.0</version>
+			<version>3.0.1</version>
```
Comment (dmatsev) [inline pom.xml:60]: update this from dev branch

File: src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java [hunk 2]
```diff
@@ -15,59 +15,69 @@
 /**
  * Created by Никита on 27.12.2016.
  */
-public class US_FilterEventsOnTheMapByCategoriesWithChrome {
+    public class US_FilterEventsOnTheMapByCategoriesWithChrome {
```
Comment (dmatsev) [inline src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java]: format all document please

File: src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java [hunk 3]
```diff
@@ -15,59 +15,69 @@
...
+        }
 
-    @AfterMethod
-    public void quitDriver() throws InterruptedException {
-        Thread.sleep(2000);
-        driver.quit();
-    }
+        @AfterMethod
+        public void quitDriver() throws InterruptedException {
+            Thread.sleep(2000);
```
Comment (dmatsev) [inline src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java]: why we need it?

File: src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java [hunk 4]
```diff
@@ -15,59 +15,69 @@
...
-        if (element == null){
-            return false;
+        public void setUpPreconditions(){
+            driver.get(mapLink);
+            categoriesDropdownList = wait.until(ExpectedConditions.presenceOfElementLocated(categoriesLocator));
+            categoriesDropdownList.click();
+            eventsDropdownList = wait.until(ExpectedConditions.presenceOfElementLocated(eventsLocator));
+            eventsDropdownList.click();
+            eventsName = wait.until(ExpectedConditions.presenceOfElementLocated(eventsNameLocator));
+
```
Comment (dmatsev) [inline src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java]: remove empty line

File: src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java [hunk 5]
```diff
@@ -15,59 +15,69 @@
...
+        }
+
+        public boolean exists(WebElement element){
+            if (element == null){
```
Comment (dmatsev) [inline src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java]: I don't understand this logic. Explain please

**Review by dmatsev — CHANGES_REQUESTED** [review]
The same comments are valid for Firefox

**Review by MaxKovalets — APPROVED** [review]

**Pull request merged**